src/Microsoft.DotNet.Tools.Publish/PublishCommand.cs

```diff
@@ -14,124 +14,74 @@
 
 namespace Microsoft.DotNet.Tools.Publish
 {
-    public class Program
+    public class PublishCommand
     {
-        public static int Main(string[] args)
-        {
-            DebugHelper.HandleDebugSwitch(ref args);
+        public string ProjectPath;
+        public string Configuration;
+        public string OutputPath;
+        public string Framework;
+        public string Runtime;
+        public NuGetFramework NugetFramework;
+        public IEnumerable<ProjectContext> ProjectRidFxPermutations;
```

Comment:
Some good reasons:
http://blogs.msdn.com/b/vbteam/archive/2009/09/04/properties-vs-fields-why-does-it-matter-jonathan-aneja.aspx

We should never set ourselves up to face our customers with breaking changes. Yes, NuGet packaging does give our customers some protection and recourse against breaking changes. Even so, we shouldn't strive to break compiled code for the sake of a few characters... particularly when doing so means discarding a well known best practice.